src/kit/taosdemo/taosdemo.c

```diff
@@ -2663,117 +2663,130 @@ static int printfInsertMeta() {
+                        g_Dbs.db[i].superTbls[j].timeStampStep);
+                printf("      startTimestamp:    \033[33m%s\033[0m\n",
+                        g_Dbs.db[i].superTbls[j].startTimestamp);
+                printf("      sampleFormat:      \033[33m%s\033[0m\n",
+                        g_Dbs.db[i].superTbls[j].sampleFormat);
+                printf("      sampleFile:        \033[33m%s\033[0m\n",
+                        g_Dbs.db[i].superTbls[j].sampleFile);
+                printf("      tagsFile:          \033[33m%s\033[0m\n",
+                        g_Dbs.db[i].superTbls[j].tagsFile);
+                printf("      columnCount:       \033[33m%d\033[0m\n        ",
+                        g_Dbs.db[i].superTbls[j].columnCount);
+                for (int k = 0; k < g_Dbs.db[i].superTbls[j].columnCount; k++) {
+                    //printf("dataType:%s, dataLen:%d\t", g_Dbs.db[i].superTbls[j].columns[k].dataType, g_Dbs.db[i].superTbls[j].columns[k].dataLen);
+                    if ((0 == strncasecmp(g_Dbs.db[i].superTbls[j].columns[k].dataType,
+                                    "binary", 6))
+                            || (0 == strncasecmp(g_Dbs.db[i].superTbls[j].columns[k].dataType,
+                                    "nchar", 5))) {
+                        printf("column[%d]:\033[33m%s(%d)\033[0m ", k,
+                                g_Dbs.db[i].superTbls[j].columns[k].dataType,
+                                g_Dbs.db[i].superTbls[j].columns[k].dataLen);
+                    } else {
+                        printf("column[%d]:\033[33m%s\033[0m ", k,
+                                g_Dbs.db[i].superTbls[j].columns[k].dataType);
+                    }
+                }
+                printf("\n");
+
+                printf("      tagCount:            \033[33m%d\033[0m\n        ",
+                        g_Dbs.db[i].superTbls[j].tagCount);
+                for (int k = 0; k < g_Dbs.db[i].superTbls[j].tagCount; k++) {
+                    //printf("dataType:%s, dataLen:%d\t", g_Dbs.db[i].superTbls[j].tags[k].dataType, g_Dbs.db[i].superTbls[j].tags[k].dataLen);
+                    if ((0 == strncasecmp(g_Dbs.db[i].superTbls[j].tags[k].dataType,
+                                    "binary", strlen("binary")))
+                            || (0 == strncasecmp(g_Dbs.db[i].superTbls[j].tags[k].dataType,
+                                    "nchar", strlen("nchar")))) {
+                        printf("tag[%d]:\033[33m%s(%d)\033[0m ", k,
+                                g_Dbs.db[i].superTbls[j].tags[k].dataType,
+                                g_Dbs.db[i].superTbls[j].tags[k].dataLen);
+                    } else {
+                        printf("tag[%d]:\033[33m%s\033[0m ", k,
+                                g_Dbs.db[i].superTbls[j].tags[k].dataType);
+                    }
                 }
+                printf("\n");
             }
-            printf("\n");
         }
+        else
```

Comment:
please keep the same coding style as following:

if (cond) {
    do something
} else {
   do something else
}